data/mods/Aftershock/mutations/mutations.json

```diff
@@ -382,7 +381,7 @@
     "description": "Your skin has patches of light fur.  This has no impact on your life except marking you as not fully human.",
     "types": [ "SKIN" ],
     "changes_to": [ "LIGHTFUR" ],
-    "category": [ "MASTODON" ]
+    "category": [ "MASTODON", "SPIDER", "MOUSE", "BEAST", "CATTLE", "RAT", "FELINE", "LUPINE", "RABBIT", "INSECT", "URSINE" ]
```

Comment:
would it be best to extend to Mastodon here so we don't have any load order overwrite issues?

Reply:
No, because this mutation is introduced by Aftershock and not present in base DDA.

Reply:
That makes sense. That probably also means I made that specifically to make mastodon pass the old threshold tests. I’ll try and look at it and see if it makes sense anymore after your changes are merged. :)